themes/theme-ios11/pages/Scanner/index.jsx

```diff
@@ -42,17 +42,15 @@ class ScannerView extends PureComponent {
       <View background={backgroundColor}>
         <RouteContext.Consumer>
           {({ query: { scope = SCANNER_SCOPE_DEFAULT, type = SCANNER_TYPE_BARCODE } = {} }) => (
-            <Fragment>
+            <ScannerContainer
+              scope={scope}
+              type={type}
+              scannerDidOpen={this.removeBackground}
+              scannerDidClose={this.resetBackground}
+            >
               <BackBar title="titles.scanner" right={null} />
```

Comment:
To make the `BackBar` customisable it should be surrounded with it's own portals or put the `BackBar` and the `ScannerContainer` next to each other inside a `scanner` portal.

Reply:
We will not customize BackBar